stdlib/2.7/__builtin__.pyi

```diff
@@ -879,3 +880,15 @@ class file(BinaryIO):
     def write(self, data: str) -> None: ...
     def writelines(self, data: Iterable[str]) -> None: ...
     def truncate(self, pos: int = ...) -> int: ...
+
+def apply(object: object, *args: Iterable, **kwargs: Dict[str, Any]) -> Any: ...
+
+def coerce(x: _Number, y: _Number) -> Tuple[_Number,_Number]: ...
```

Comment:
Sadly this doesn't work -- mypy doesn't appear to consider the concrete numeric types (int, float etc.) to be instances of Number.

Maybe a type variable would work?
```
_N = TypeVar('_N', int, float, complex)
def coerce(x: _N, y: _N) -> Tuple[_N, _N]: ...
```
???